bears/vcs/git/GitCommitBear.py

```diff
@@ -20,46 +21,33 @@ def check_prerequisites(cls):
         else:
             return True
 
-    def run(self,
-            shortlog_length: int=50,
-            body_line_length: int=72,
-            force_body: bool=False,
-            allow_empty_commit_message: bool=False,
-            shortlog_regex: str="",
-            shortlog_trailing_period: bool=None,
-            shortlog_imperative_check: bool=True,
-            shortlog_wip_check: bool=False):
+    @classmethod
+    def get_shortlog_checks_metadata(cls):
+        return FunctionMetadata.from_function(
+            cls.check_shortlog,
+            omit={"self", "shortlog"})
+
+    @classmethod
+    def get_body_checks_metadata(cls):
+        return FunctionMetadata.from_function(
+            cls.check_body,
+            omit={"self", "body"})
+
+    @classmethod
+    def get_metadata(cls):
+        return FunctionMetadata.merge(GlobalBear.get_metadata(),
```

Comment:
`run()` contains also settings, that's why I want to use the old "behaviour" too